### ISSUE TITLE
Create for PostgreSQL 

### ISSUE THREAD

**Issue by edeunix** [mevdschee/php-crud-api#107]
Hi!

First of all, thank you to share with us, and congratulations.

I am using the PHP-API-CRUD on my PostgreSQL. The functions of Read, Update and Delete works fine, but Create (POST method) has a bug, always returning 'null'.
I'm trying to fix it, putting  debug output variables in the code, but at this moment I do not see any problem on the code.
I'm thinking to deploy my database at a MySQL too, and verify if this problem is only in PostgreSQL INPUT query.

Best,
--E

**Comment by mevdschee** [comment]
Thank you for reaching out and for your compliments.

Did you run the tests? Do they succeed? Please provide the smallest test case in which it fails. Things that would help are a database structure dump and the url your are trying to post to (and the parameters/data).

**Comment by mevdschee** [comment]
On my box the tests (including a post method test) run fine: 

```
maurits@nuc:~/projects/php-crud-api$ php phpunit.phar tests/tests.php 
PHPUnit 5.3.2 by Sebastian Bergmann and contributors.

.............................................                     45 / 45 (100%)

Time: 949 ms, Memory: 8.00Mb

OK (45 tests, 69 assertions)
maurits@nuc:~/projects/php-crud-api$ 
```

What are your findings?

**Comment by edeunix** [comment]
Hi @mevdschee 

I will run the tests using postgres, but I tried to use the API changing my dbengine to the MySQL, and its works!
Other thing what I saw is the params in POST method were sent, both to json and unencoded mime, but maybe the an error occurs in the conversion of this to the INSERT postgre syntax. 
First of all, I need to run the tests for postgre on my local testbed =). All of this are only assumptions, and now your API and MySQL works for me, and it is great!
